stable/spring-cloud-data-flow/values.yaml

```diff
@@ -16,19 +16,45 @@ initContainers:
     tag: "1.30.1"
     imagePullPolicy: IfNotPresent
 
+nameOverride: ""
```

Comment:
all of these added values need to also be defined in the README per the chart req's